<ui version="4.0" >
 <class>CalibrationSetupsDialog</class>
 <widget class="QDialog" name="CalibrationSetupsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Hardware Setups</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QPushButton" name="remove" >
     <property name="toolTip" >
      <string>Remove the currently selected hardware setup and all its calibrations</string>
     </property>
     <property name="text" >
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="add" >
     <property name="toolTip" >
      <string>Add an empty hardware setup with the entered name</string>
     </property>
     <property name="text" >
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QPushButton" name="exportSetup" >
     <property name="toolTip" >
      <string>Export the currently selected hardware setup with all its calibrations to a file</string>
     </property>
     <property name="text" >
      <string>Export...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="rename" >
     <property name="toolTip" >
      <string>Rename the currently selected hardware setup</string>
     </property>
     <property name="text" >
      <string>Rename</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLineEdit" name="input" />
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="importSetup" >
     <property name="toolTip" >
      <string>Import a hardware setup and its calibrations from a file</string>
     </property>
     <property name="text" >
      <string>Import...</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="0" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="list" />
     </item>
     <item>
      <widget class="QListWidget" name="detailsList" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>input</tabstop>
  <tabstop>list</tabstop>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>rename</tabstop>
  <tabstop>exportSetup</tabstop>
  <tabstop>importSetup</tabstop>
  <tabstop>detailsList</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalibrationSetupsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalibrationSetupsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
